operations/network/management_network/bican_technical_details.md

```diff
@@ -0,0 +1,249 @@
+    5. [Traffic separation and routing](#25-traffic-separation-and-routing)
+        1. [Touchpoints: effects and changes](#251-touchpoints-effects-and-changes)
+        2. [When configuration occurs](#252-when-configuration-occurs)
+        3. [Ability to change post-install](#253-ability-to-change-post-install)
+3. [Management CAN (CMN)](#3-custeomer-management-network-cmn)
+    1. [Traffic separation and routing](#31-traffic-separation-and-routing)
+    2. [Endpoint naming](#32-endpoint-naming)
+    3. [Endpoint addressing](#33-endpoint-addressing)
+    4. [Changes](#34-changes)
+        1. [Touchpoints: effects and changes](#341-touchpoints-effects-and-changes)
+        2. [When configuration occurs](#342-when-configuration-occurs)
+        3. [Ability to change post install](#343-ability-to-change-post-install)
+4. [CAN external/site access (site DNS, LDAP, etc.)](#4-can-externalsite-access-site-dns-ldap-etc)
+    1. [Traffic separation and routing](#41-traffic-separation-and-routing)
+    2. [Changes](#42-changes)
+        1. [Touchpoints: effects and changes](#421-touchpoints-effects-and-changes)
+        2. [When configuration occurs](#422-when-configuration-occurs)
+        3. [Ability to change post-install](#423-ability-to-change-post-install)
+
+## 1 CAN new features overview
+
+Bifurcation or splitting of the Customer Access Network (CAN) enables customization of customer traffic to and from the system.
+Customization will be performed during installation.
+For CSM 1.2 there are two new customer access networks being introduced as part of the process to split the existing monolithic CAN:
+
+1. **High Speed Customer Access - CHN (Customer High Speed Network)** : This feature adds the ability to connect to User Application Nodes (UAN), UAI, Compute Nodes
+and Kubernetes API endpoints from the customer site via the High Speed Network (HSN).
+2. **Management Customer Access - CMN (Customer Management Network)** :  Using a new VLAN on the Management Network, this feature allows system administrative access from the customer site.
+Administrative access was previously available on the original CAN; this feature provides a traffic path and access split.
+
+Enabling BICAN will remove the original CAN.
+
+**During installation, the opportunity to enable the new features will be presented.**
+
+At this time, the customers must accept:
+
+- External administrator access to administrative endpoints moves to the Customer Management Network (CMN).
+- External user access to user endpoints moves to the Customer High-speed Network (CHN).
+- The CHN is now the default ingress network.
+- Customer access over the legacy CAN is disabled by default.
+
+Reverting or changing any decisions will be manual.
+
+For the feature matrix of the CAN, see [BICAN Support Matrix](bican_support_matrix.md).
+Details of the High Speed CAN (CHN) and the Management CAN (CMN) are described below.
```

Comment:
The spelled out versions of CHN and CMN vary a across these files. We should keep this consistent.

Reply:
did a pass at making CHN and CMN more consistent